clients/include/utility.hpp

```diff
@@ -912,6 +912,122 @@ hipsparseStatus_t host_nnz(hipsparseDirection_t      dirA,
     return HIPSPARSE_STATUS_SUCCESS;
 }
 
+template <hipsparseDirection_t DIRA, typename T>
+hipsparseStatus_t host_dense2csx(int                  m,
```

Comment:
This is interesting. I see that you are returning a status. We don't do that in the other host utility calls but maybe we should?

Reply:
Maybe we should ... that's a good point. It seems that I didn't follow the convention.
But since I see that I'm not using the returned status, I made the host routines returning void instead.
But maybe we should ...